src/index.html

```diff
@@ -7,28 +7,61 @@ <h1>Instant, themed Twitch chat overlays</h1>
 	<form id="overlay-builder">
 		<label for="channel">Channel name</label>
 		<input type="text" id="channel" name="channel" required="true" autocomplete="off" autocorrect="off"
-			autocapitalize="off" spellcheck="false" />
-		<label for="theme">Theme</label>
+			autocapitalize="off" spellcheck="false" placeholder="{{defaults.channel}}"
+			title="Your Twitch channel name" />
+
+		<label for="theme">Theme <a href="docs/configuration/#theme">?</a></label>
 		<select id="theme" name="theme">
-			<option value="default">Default</option>
+			<option value="{{defaults.theme}}">{{defaults.theme | formatThemeName}}</option>
 			<option value="light">Light</option>
 			{%- for theme in themes.list -%}
-			{%- unless theme == "default" or theme =="light" -%}
+			{%- unless theme == "default" or theme == "light" -%}
 			<option value="{{ theme }}">{{ theme | formatThemeName }}</option>
-			{%- endunless -%} {%- endfor %}
+			{%- endunless -%}
+			{%- endfor %}
+		</select>
+
+		<label for="demo">Demo page <a href="docs/configuration/#demo">?</a></label>
+		<select id="demo" name="DEMO">
+			<option>{{defaults.DEMO}}</option>
+			<option>true</option>
```

Comment:
The reason the `DEMO` params is formatted differently from the other params is that I anticipated it only being used internally for the preview mode. So... I'm not sure whether we should have it in the URL builder. I'm open to thoughts, however, and I'd love for folks to weigh in!